node_io/test/file_test.dart

```diff
@@ -27,6 +28,36 @@ void main() {
       expect(file.existsSync(), isTrue);
     });
 
+    test('exists', () async {
+      expect(await new File(join(Directory.current.path, "__dummy__")).exists(),
```

Comment:
Similarly a helper function for creating a File would make these tests a bit cleaner.

Reply:
ok